lib/Mojo/Loader.pm

```diff
@@ -15,12 +15,17 @@ sub data_section { $_[0] ? $_[1] ? _all($_[0])->{$_[1]} : _all($_[0]) : undef }
 sub file_is_binary { keys %{_all($_[0])} ? !!$BIN{$_[0]}{$_[1]} : undef }
 
 sub find_modules {
-  my $ns = shift;
+  my ($ns, $options) = (shift, shift // {});
+
+  my @ns  = split '::', $ns;
```

Comment:
`split /::/, $ns;`

Reply:
We actually use the string form everywhere in Mojolicious.

Reply:
Weird, I'll suggest changing that separately...